### PR TITLE
Auto-update the dotnet SDK

### PR DESCRIPTION
**Pull request by kfcampbell**
This PR uses the `gr2m/create-or-update-pull-request-action@cd-path` Action to autogenerate commits to the octokit/dotnet-sdk repo.

### CODE REVIEW

**Comment by kfcampbell** [comment]
@nickfloyd do you mind sharing the versions of .NET, Go, and Kiota you used to create the initial commit of the dotnet-sdk? I can reproduce these build errors locally.

**Comment by kfcampbell** [comment]
Ahh, I got it working locally. Builds should succeed with a version bump for Kiota's helpers, done in https://github.com/octokit/dotnet-sdk/pull/2. 

In the future, we should do something for .NET like we do for Go in post-processing, where the latest versions of the helpers are always installed. That way, the helper package versions will keep updated in lockstep with Kiota versions and we can ensure compatability.

**Comment by kfcampbell** [comment]
Woohoo! The build is now successful and https://github.com/octokit/dotnet-sdk/pull/3 has been created for the changes. I'm going to publish this PR now. 

@nickfloyd when you review, do you mind double-checking the namespacing? I may have messed something up, I'm seeing this in the diff:

```csharp
- namespace GitHub.Octokit.Advisories {
+ namespace Octokit.Client.Advisories {
```

Maybe that's expected?

File: .github/workflows/build-csharp.yml [hunk 1]
```diff
@@ -17,30 +17,50 @@ jobs:
...
+        run: |
+          cd ../
+          git clone https://github.com/octokit/dotnet-sdk.git
 
       - name: Download latest schema
         run: go run schemas/main.go --schema-next=false
 
       - name: Run generation
-        run: time kiota generate -l csharp --ll trace -o generated/csharp/Octokit -c OctokitClient -n Octokit.Client -d schemas/downloaded.json
+        run: time kiota generate -l csharp --ll trace -o $(pwd)/../dotnet-sdk/GitHub/Octokit -c OctokitClient -n Octokit.Client -d schemas/downloaded.json
```
Comment (nickfloyd) [inline .github/workflows/build-csharp.yml:40]:
```suggestion
        run: time kiota generate -l csharp --ll trace -o $(pwd)/../dotnet-sdk/GitHub/Octokit -c OctokitClient -n GitHub.Octokit -d schemas/downloaded.json
```

**Pull request merged**